apps/src/templates/BaseDialog.jsx

```diff
@@ -7,6 +7,9 @@ import {BASE_DIALOG_WIDTH} from '../constants';
  * A generic modal dialog that has an x-close in the upper right, and a
  * semi-transparent backdrop. Can be closed by clicking the x, clicking the
  * backdrop, or pressing esc.
+ *
+ * To use a styled version of this component that follows our style guide,
+ * use StylizedBaseDialog.jsx.
  */
 export default class BaseDialog extends React.Component {
```

Comment:
Should this become `__DeprecatedBaseDialog`?

Reply:
Oops. Nevermind. On that note, is there a way we can encourage use of StylizedBaseDialog?

Reply:
Or discourage use of `BaseDialog` except where appropriate?

Reply:
yes to both latter questions -- i left [a PSA in #apps-dev](https://codedotorg.slack.com/archives/C0T0PNR0D/p1622653630005300) directing folks to use this dialog where possible, and i'm going to demo/PSA at next week's all-team demos